src/views/Map/Map.tsx

```diff
@@ -90,11 +90,6 @@ export function MapPage() {
         initEditableLayer(),
       );
     }
-    if (userModel?.hasFeature(AppFeature.INTERCOM)) {
-      import('~features/intercom').then(({ initIntercom }) => {
-        initIntercom();
```

Comment:
Do we just not init intercom at all or is it initialized in some other place?

Reply:
com inits in CommonView, this one is duplication after merges